src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -40,7 +40,7 @@ class OrchestratorCli(orchestrator.OrchestratorClientMixin, MgrModule):
             "perm": "r"
         },
         {
-            'cmd': "orchestrator osd add "
+            'cmd': "orchestrator osd create "
                    "name=svc_arg,type=CephString ",
```

Comment:
I think that we agree to use only one optional argument with  name "drive_group"  to include the same information that we can specify in the json file ( -i )

Reply:
see above